src/javascript/app_2/components/elements/drawer/toggle_drawer.jsx

```diff
@@ -10,7 +10,7 @@ class ToggleDrawer extends PureComponent {
         const { alignment } = this.props;
         if (alignment && alignment === 'left') {
```

Comment:
I think for all of these we can just check `if  (alignment === '[...]')`, no need to check if alignment exists, since if it's undefined, `undefined === 'left'` will just return false as expected